GOVERNANCE.md

```diff
@@ -0,0 +1,130 @@
+
+- Displaying a high level of commitment to the project and its community, and
+being a role model for others.
+- Writing patches &mdash; a lot of patches, especially "glue code" or "grunt
+work" or general "housekeeping"; fixing bugs, ensuring documentation is always
+high quality, consistent UX design, improving processes, making judgments on
+dependencies, handling security vulnerabilities, and so on and so forth.
+- Reviewing code submitted by others &mdash; with an eye to maintainability,
+performance, code quality, and "style" (fitting in with the project).
+- Participating in design discussions, especially with regards to architecture
+or long-term vision.
+- When necessary, participate in a collective voting process to resolve
+conflicts and make decisions for which there is no clear answer.
+
+This is not an exhaustive list, nor is it intended that every maintainer does
+each and every one of these individual tasks to equal amounts. Rather this is
+only a guideline for what maintainers are expected to conceptually do.
+
+In short, Maintainers are the outwardly visible stewards of the project.
+
+#### Current list of maintainers
+
+The current list of maintainers:
+
+- Yuya Nishihara
+- Martin von Zweigbergk
+- Waleed Khan
+- Ilya Grigoriev
+- Austin Seipp
+
+### Contributors
+
+We consider contributors to be active participants in the project and community
+who are *not* maintainers. These are people who might:
+
+- Help users by answering questions
+- Participating in lively and respectful discussions across various channels
+- Submit high-quality bug reports
+- Submit patches or pull requests
+- Help with testing and quality assurance
+- Submit feedback about planned features, use cases, or bugs
+
+We essentially define them as **people who actively participate in the
+project**. Examples of things that would *not* make you a contributor are:
+
+- Submitting a single bug report and never returning
+- Writing blog posts or other evangelism
+- Using the software in production
+- Forking the project and maintaining your own version
+
+While these are all generally quite valuable, they don't directly contribute to
+the existing community of users where they are at, and on their own do not
+constitute "active participation".
+
+Contributors and their input are valued and their input is expected to be taken
+and respected by Maintainers; not every discussion will result in a change, but
+it is intended that every voice will be heard.
+
+## Processes
+
+For the purposes of making decisions across the project, the following processes
+are defined.
+
+### Decision-Making
+
+The person proposing a decision to be made (i.e. technical, project direction,
+etc) can offer a proposal, along with a 2-to-4 week deadline for discussion.
+During this time, Maintainers may participate with a vote of:
+
+A) Support
+B) Reject
+C) Abstain
+
+A final decision requires more than half of the participating votes (i.e.
+excluding abstaining votes.)
```

Comment:
```suggestion
A final decision requires more than half of the participating votes (i.e.
excluding abstaining votes).
```

suggestion: When reading this with fresh eyes, I thought it was referring to a kind of 'quorum', i.e. you need to have X non-abstaining votes in order for the vote to be valid. I think that's not what we intended here. I think we should say explicitly something like

> Each Maintainer gets one vote. The total number of "participating votes" is the number of Maintainer votes which are not Abstain. The proposal is accepted when more than half of the participating votes are Support.

And, if necessary, we can give examples of abstention or early-exiting the decision to make it clear what we intended here.